src/js/BookReader.js

```diff
@@ -302,12 +302,58 @@ BookReader.prototype.initParams = function() {
     }
   }
 
+  // Check for Search plugin
+  if (this.options.enableSearch) {
+    // If initialSearchTerm not set
+    if (!this.options.initialSearchTerm) {
+      // Look for any term in URL
+      if (params.search) {
+        // Old style: /search/[term]
+        this.options.initialSearchTerm = params.search;
+      } else {
+        // If we have a query string: q=[term]
+        const searchParams = new URLSearchParams(this.readQueryString());
```

Comment:
Q: is there a reason why we aren't adding it to `params.search` ?

Reply:
This is an init case, don't need it. options.initialSearchTerm is what's needed
This particular line creates a URLSearchParams object